TestProjects/Playground/Playground.Forms.Wpf/Playground.Forms.Wpf.csproj

```diff
@@ -54,7 +54,6 @@
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
     </ApplicationDefinition>
-    <Compile Include="Setup.cs" />
```

Comment:
I'm not sure why this file is being removed as it's not currently part of the WPF forms project - can you confirm that you've rebased off latest Develop

Reply:
I have rebased off the latest develop, and this file isn't it repository anymore, however still in csproj and so the TestProjects build failed.